lib/srv/authhandlers.go

```diff
@@ -387,6 +387,8 @@ func (h *AuthHandlers) canLoginWithoutRBAC(cert *ssh.Certificate, clusterName st
 // client) to see if this certificate can be allowed to login as user:login
 // pair to requested server and if RBAC rules allow login.
 func (h *AuthHandlers) canLoginWithRBAC(cert *ssh.Certificate, clusterName string, teleportUser, osUser string) error {
+	ctx := context.TODO()
```

Comment:
It might be preferable to use `h.c.Server.Context()` instead of `context.TODO()`.